docker-compose-elastic-cloud.yml

```diff
@@ -0,0 +1,125 @@
+version: "2.1"
+services:
+  opbeans-node:
+    build: .
+    image: opbeans/opbeans-node:latest
+    ports:
+    - "127.0.0.1:${OPBEANS_SERVER_PORT:-8000}:3000"
+    logging:
+      driver: 'json-file'
+      options:
+        max-size: '2m'
+        max-file: '5'
+    environment:
+      - ELASTIC_APM_SERVICE_NAME=${ELASTIC_APM_SERVICE_NAME:-opbeans-node}
```

Comment:
This line is redundant... the agent will use the name in `package.json` automatically (which is `opbeans-node` in our case), or if `ELASTIC_APM_SERVICE_NAME` is set, it will use that instead

Reply:
https://github.com/elastic/opbeans-node/pull/60#discussion_r323178398